index.html

```diff
@@ -130,7 +130,26 @@ <h3 class="card-title">Age of Bliss</h3>
         <span id="exit" class="hide">Less<i class="fas fa-angle-up"></i></span>
       </button>
     </div>
-
+  
+    <!-- Footer Section -->
+    <div class="partner-section">
+      <div class="partner-container">
+        <h2>Partner</h2>
+        <hr class="solid">
+        <div class="partner-logos">
+          <img class="heineken-logo" src="images/heineken.png" alt="Heineken">
+          <img class="cocacola-logo" src="images/coca-cola.png" alt="CocaCola">
+          <img class="samsung-logo" src="images/samsung.png" alt="Samsung">
+          <img class="spotify-logo" src="images/spotify.png" alt="Spotify">
+        </div>
+      </div>
+    </div>
+    <footer class="footer-section">
+      <div class="footer-container">
+        <img src="images/nyeusi-gray.png" alt="Nyeusi Music Festival" class="footer-logo">
+        <p class="index-footer-text">2022 Nyeusi Music Festival <i class="far fa-registered"></i> Some rights reserved.</p>
+      </div>
+    </footer>
     <script src="javascript/acts.js"></script>
     <script src="javascript/menu.js"></script>
   </body>
```

Comment:
Your mobile hamburger button is on the left side and it works like a charm 😎 Cool and smooth transition 🙌

Reply:
Great use of semantics 😎 The proper use of tags is key to having better HTML! Well done using `section`, `article`, and `footer` tags 💪